### ISSUE TITLE
Add details in challenge phase view

### ISSUE THREAD

**Issue by deshraj** [Cloud-CV/EvalAI#704]
This issue involves solving following issues:

- [ ] Format start date and end date on challenge phase view. For reference, see this line https://github.com/Cloud-CV/EvalAI/blob/master/frontend/src/views/web/challenge/challenge-list.html#L17

- [ ] Show `Maximum number of submissions per day` and `Maximum number of total submissions` in the card

**Comment by jainarunarunjain** [comment]
@deshraj Can you assign this to me.

**Comment by deshraj** [comment]
Sure, assigning it to you.

**Comment by aka-jain** [comment]
@deshraj Where do you want to show the submission count?

**Comment by deshraj** [comment]
I think we should show it before the description of every phase. For reference, see this http://evalai.cloudcv.org/web/challenges/challenge-page/1/phases

**Comment by aka-jain** [comment]
Okay, do we also need to show a warning if user tries to exceed the submission count?

**Comment by deshraj** [comment]
I think we can open a different issue for that. It would be a cool feature to have. We would need to write a new API endpoint for that.

**Comment by aka-jain** [comment]
Agree. 
@jainarunarunjain You just need to add both field to phase details only. Let me know if you need more help.

**Comment by jainarunarunjain** [comment]
@aka-jain @deshraj Sure, I will do this.

**Comment by jainarunarunjain** [comment]
@deshraj @aka-jain From which API do i get max submission count.

**Comment by aka-jain** [comment]
Ammm, don't know. @deshraj Is that count not coming in phase API?

**Comment by jainarunarunjain** [comment]
@deshraj I need some help, 
max submissions count is not coming in phase API.

**Comment by RishabhJain2018** [comment]
@deshraj I think we have not included the `max_submissions_per_day` and `max_submissions` field in the 
`ChallengePhaseSerializer`. Please refer to this [link](https://github.com/Cloud-CV/EvalAI/blob/master/apps/challenges/serializers.py#L33). What are your views about it?

**Comment by deshraj** [comment]
@RishabhJain2018 it would be great if you can add these fields in the API so that we can unblock @jainarunarunjain. I am occupied with some other work.

**Comment by RishabhJain2018** [comment]
@deshraj yeah sure. I think I should open a new issue for that?

**Comment by deshraj** [comment]
Cool. Sounds good. And don't forget to reference to this issue.

**Comment by RishabhJain2018** [comment]
okay sure.

**Comment by RishabhJain2018** [comment]
@jainarunarunjain The challenge Phase API is now updated. Now it is returning `max_submissions_per_day` and `max_submissions`.

**Comment by jainarunarunjain** [comment]
Thnx @RishabhJain2018 @deshraj

**Comment by jainarunarunjain** [comment]
@deshraj Fixed with #729

**Comment by aka-jain** [comment]
Closing it. Fixed with #729.
